packages/mojatax/src/Utils.js

```diff
@@ -0,0 +1,87 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  assertRelativeUrl,
+  logResponse,
+} from '@openfn/language-common/util';
+import nodepath from 'node:path';
+
+export const authorize = state => {
+  const auth = state.configuration;
+  if (auth.access_token) {
+    return state;
+  }
+  const clientId = auth.clientId;
```

Comment:
Little tip: if you do:
```
const { clientId: clientId, password } = auth;
```

Then your body declaration can use shorthand properties names:

```
const body = { client_id, password }
```

And because that's so simple, you can just declare the body inline on the options and don't need to declare the extra variable